test/builder/lib/QuarkBuilderTest.sol

```diff
@@ -9,8 +9,11 @@ import {CodeJarHelper} from "src/builder/CodeJarHelper.sol";
 import {Paycall} from "src/Paycall.sol";
 import {PaymentInfo} from "src/builder/PaymentInfo.sol";
 import {QuarkBuilder} from "src/builder/QuarkBuilder.sol";
+import {Strings} from "src/builder/Strings.sol";
 
 contract QuarkBuilderTest {
+    uint256 constant BLOCK_TIMESTAMP = 123_456_789;
+
     address constant COMET_1 = address(0xc3);
 
     address constant LINK_1 = address(0xfeed01);
```

Comment:
iirc XXX_# the # means the chain id. 
To prevent future confusion, probably it's better to use the actual addresses for those that are marked with the actual chain Id like 1 as mainnet?